pkgs/by-name/ol/ollama/package.nix

```diff
@@ -87,10 +88,17 @@ let
   ];
   cudaToolkit = buildEnv {
     name = "cuda-merged";
-    paths = map lib.getLib cudaLibs ++ [
-      (lib.getOutput "static" cudaPackages.cuda_cudart)
-      (lib.getBin (cudaPackages.cuda_nvcc.__spliced.buildHost or cudaPackages.cuda_nvcc))
-    ];
+    paths =
+      map lib.getLib [
+        cudaPackages.cuda_cudart
+        cudaPackages.cuda_cudart.stubs
+        cudaPackages.libcublas
+        cudaPackages.cuda_cccl.dev
```

Comment:
When I checked the `/nix/store`, I noticed there's no lib directory under cuda_cccl or cuda_cudart. That's why I'm using `cuda_cccl.dev` and `cuda_cccl.stubs` instead. Did I misunderstand the purpose of `lib.getLib` here?